### PR TITLE
Revert "[DOCS-6692] - Remove unwanted process sentence"

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
Reverts Alfresco/docs-alfresco#613

### CODE REVIEW

**Review by Artur-Piechuta — APPROVED** [review]
Looks fine for me 👍

**Review by anxumalo — DISMISSED** [review]
Added a few minor issues - feel free to add them to a future JIRA

File: process-automation/latest/using/index.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ To start a process instance:
 
     **Note:** Once you have selected the process you would like to work with from the modal window you can copy and share the URL with another user, so they can start the process. This user must have credentials to sign into the Digital Workspace.
 
+All Alfresco Digital Workspace users who have initiated a process or participated in one can see all the tasks from the process, including the tasks assigned to other users. To see the list of tasks navigate to `Process Management` > `Processes` > `Running` > `Process details`.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/using/index.md:59]: Done corrected

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**